fluidkeys/main.go

```diff
@@ -238,6 +247,25 @@ func printSecretKeyListing(key gpgwrapper.SecretKeyListing) string {
 	return output
 }
 
+func promptForKeyToImportFromGpg(secretKeyListings []gpgwrapper.SecretKeyListing) gpgwrapper.SecretKeyListing {
+	var selectedKey int
+	for validInput := false; !validInput; {
+		rangePrompt := colour.LightBlue(fmt.Sprintf("[1-%v]", len(secretKeyListings)))
+		input := promptForInput(fmt.Sprintf(PromptWhichKeyFromGPG + " " + rangePrompt + " "))
+		if intergerSelected, err := strconv.Atoi(input); err != nil {
+			fmt.Printf("Only numbers please\n")
```

Comment:
Ditto my comment about "Number not available", it's strange feeling error message :)

Reply:
Yeah, prefer your suggestion. Will implement that. I think printing the keys again is a bit much given they're just a line or two above.